homeassistant/components/nissan_leaf/device_tracker.py

```diff
@@ -15,17 +15,14 @@
 
 def setup_scanner(hass, config, see, discovery_info=None):
     """Set up the Nissan Leaf tracker."""
-    _LOGGER.debug("Setting up Scanner (device_tracker) for Nissan Leaf, "
-                  "discovery_info=%s", discovery_info)
-
     def see_vehicle():
         """Handle the reporting of the vehicle position."""
         for key, value in hass.data[DATA_LEAF].items():
```

Comment:
Clarify variable names as above.

Reply:
Fixed in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/daca6ede4711c75f55d971aed1acdf3311c4553c